src/CSharpMongoMigrations/MigrationRunner.cs

```diff
@@ -16,17 +17,26 @@ public class MigrationRunner
         /// <summary>
         /// Cts
         /// </summary>
-        /// <param name="server">MongoDb server connection string</param>
-        /// <param name="database">Mongo database name</param>
+        /// <param name="url">MongoDb connection string</param>
         /// <param name="migrationAssembly">Assembly with migrations</param>
-        public MigrationRunner(string server, string database, string migrationAssembly)
```

Comment:
You shouldn't remove the old constructor because it breaks a backward compatibility.
You need to create a new one in your case.

Reply:
Restored the old constructor.